forum/projects/models.py

```diff
@@ -1,3 +1,44 @@
+from django.core.validators import MinValueValidator
 from django.db import models
+from profiles.models import StartupProfile
 
 # Create your models here.
+
+class Project(models.Model):
+    startup = models.ForeignKey(StartupProfile, on_delete=models.CASCADE, related_name="project")
+    title = models.CharField(max_length=100)
+    funding_goal = models.DecimalField(
+        max_digits=15, 
+        decimal_places=2, 
+        validators=[MinValueValidator(0.00)]
+        )
```

Comment:
`funding_goal` has a minimum value validator but no explicit database constraints.
Add a database-level `CheckConstraint` to enforce non-negative values:
```
class Meta:
    constraints = [
        models.CheckConstraint(
            check=models.Q(funding_goal__gte=0),
            name="funding_goal_non_negative"
        )
    ]
```

Reply:
Added `CheckConstraint` in `funding_goal`.